CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v.1.9.0
+
+### 28/10/2019
+
+### Features
+- Dependencies upgrades
```

Comment:
No need to specify "Dependencies upgrades" since the release note is for the customers.